src/main/scala/com/lightbend/conductr/sbt/ConductRPlugin.scala

```diff
@@ -170,6 +312,31 @@ object ConductrPlugin extends AutoPlugin {
   def sandboxStop(): Unit =
     Process(Seq("sandbox", "stop")).!
 
+  /**
+   * A convenience function that waits on ConductR to become available.
+   *
+   * @param duration the max time to wait.
+   */
+  def waitForConductr(implicit duration: FiniteDuration): Unit = {
+    print("Waiting for ConductR to start")
+
+    val deadline = duration.fromNow
+
+    @tailrec
+    def loop(deadline: Deadline): Unit = {
+      print(".")
+      if (deadline.isOverdue())
+        throw new TimeoutException(s"ConductR has not been started within ${duration.toSeconds} seconds!")
```

Comment:
The message is logged as an error at the console... which I think is better than colouring yourself. ;-)